releasenotes/notes/sbplx_optimizer-a27361fc4ac8de8c.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support for SBPLX optimizer from NLopt library has been added.
+    SBPLX is a local gradient-free optimizer based on Nelder-Mead and
+    is expected to show better convergence behaviour.
```

```suggestion
    is expected to show better convergence behavior.
```
Needs the US Spelling without the U